projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json

```
@@ -0,0 +1,176 @@
+{
+  "advancements.defeat_boss.descr": "打败一位远古神明",
+  "advancements.defeat_boss.title": "弑神者",
+  "advancements.get_burnt_by_light.descr": "站在日光射线前面",
+  "advancements.get_burnt_by_light.title": "高压危险",
+  "advancements.interact_with_mirror.descr": "与反射器交互",
+  "advancements.interact_with_mirror.title": "无线网络",
+  "advancements.kill_villager_with_macu.descr": "利用黑曜石锯剑或黑曜石匕首杀死一个村民",
+  "advancements.kill_villager_with_macu.title": "活祭血神",
```

```suggestion
  "advancements.kill_villager_with_macu.title": "血祭血神",
```
原文`Blood for the Blood God`